model/plugins/patch_user/operations.smithy

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/latest/security/access-control/api/#patch-user"
+)
+
+@vendorExtensions(
+    "x-operation-group": "security.patch_user",
```

Comment:
Since the "security.patch_user" is the name of the group, the folder structure should be similar: `model/security/patch_user/`